src/js/index.js

```diff
@@ -147,9 +148,11 @@ function create() {
 
   pretzel = new Pretzel(this, 400, 400, 1);
   hockeyStick = new Weapon(this, 320, 300, 'hockey_stick');
+  insectPart = new InsectPart(this, 500, 500, 1);
```

Comment:
do `new InsectPart(game, 500, 500, 1);`

idk who started the `this` thing but it breaks stuff